tests/unit/src/test/scala/tests/codeactions/MillifyDependencyLspSuite.scala

```diff
@@ -0,0 +1,65 @@
+package tests.codeactions
+
+import scala.meta.internal.metals.codeactions.ConvertToNamedArguments
+import scala.meta.internal.metals.codeactions.ExtractValueCodeAction
+import scala.meta.internal.metals.codeactions.MillifyDependencyCodeAction
+import scala.meta.internal.metals.codeactions.RewriteBracesParensCodeAction
+import scala.meta.internal.metals.codeactions.StringActions
+
+class MillifyDependencyLspSuite
+    extends BaseCodeActionLspSuite("millifyDependency") {
+
+  check(
+    "script-import",
+    "import $ivy.\"org.scalameta\" %%<<>> \"metals\" % \"1.0\"",
+    MillifyDependencyCodeAction.title("`org.scalameta::metals:1.0`"),
+    "import $ivy.`org.scalameta::metals:1.0`",
+    fileName = "script.sc",
+  )
+
+  check(
+    "script-import-cursor-after-import",
+    "import $ivy.\"org.scalameta\" %% \"metals\" % \"1.0\"<<>>",
+    s"""|${StringActions.multilineTitle}
+        |${StringActions.interpolationTitle}
+        |${MillifyDependencyCodeAction.title("`org.scalameta::metals:1.0`")}""".stripMargin,
+    "import $ivy.`org.scalameta::metals:1.0`",
+    fileName = "script.sc",
+    selectedActionIndex = 2,
+  )
+
+  check(
+    "mill-ivy",
+    """|import mill._, scalalib._
+       |
+       |object MyModule extends ScalaModule {
+       |  def ivyDeps = Agg(
+       |    "org.scalameta" %% "metals" % "1.0"<<>>
+       |  )
+       |}""".stripMargin,
+    s"""|${StringActions.multilineTitle}
+        |${StringActions.interpolationTitle}
+        |${RewriteBracesParensCodeAction.toBraces("Agg")}
+        |${ExtractValueCodeAction.title("\"org.scala` ...")}
+        |${ConvertToNamedArguments.title("Agg(...)")}
+        |${MillifyDependencyCodeAction.title("`org.scalameta::metals:1.0`")}
+        |${MillifyDependencyCodeAction.title("ivy\"org.scalameta::metals:1.0\"")}""".stripMargin,
```

Comment:
Should we maybe only have this action and not the one above?

Reply:
Do you have any example where it would be useful? Maybe it's ok to have just the ivy one for now.

Reply:
Wouldn't the first case in the video still be valid in a mill build file? But I think it would be a lot more uncommon to use in deps that weren't already in Mill format here (mainly mill plugins, etc), I've removed it in [c4d4cca](https://github.com/scalameta/metals/pull/4465/commits/c4d4ccab62dea6214175f3b19602c0609c095d0e) :slightly_smiling_face: